Saule/JsonApiSerializer.cs

```diff
@@ -72,7 +72,9 @@ public PreprocessResult PreprocessContent(object @object, ApiResource resource,
                     type: resource,
                     baseUrl: requestUri,
                     urlBuilder: UrlPathBuilder,
-                    paginationContext: QueryContext?.Pagination);
+                    paginationContext: QueryContext?.Pagination,
+                    includingContext: QueryContext?.Including,
+                    includedDefault: QueryContext != null ? QueryContext.IncludedDefault : true);

```

Comment:
Looking at this again there might be some confusion here. I now see that the `QueryContext` was intended for only the querystring parameters for the request. I thought about the word Query in a broader context i.e something like Request.

I added `IncludedDefault` without considering the above. Perhaps it would be better to separate the `IncludedDefault` state into another property of the `Request.Properties` of the `HttpActionContext` but then again it might not be worth that extra separation. What is your opinion @joukevandermaas?